neptune/oauth.py

```diff
@@ -66,7 +69,7 @@ def _refresh_token(self):
     def _refresh_session_token(self):
         self.session.refresh_token(self.session.auto_refresh_url, verify=self.session.verify)
         if self.session.token is not None and self.session.token.get('access_token') is not None:
-            decoded_json_token = jwt.decode(self.session.token.get('access_token'), verify=False)
```

Comment:
Make sure that this works on ALL environments and why was this given here. It's VERY easy to break authentication

Reply:
algorithm is no longer required in 2.0?